apps/community/src/app/member/professor/components/ProfessorCard.tsx

```diff
@@ -0,0 +1,49 @@
+'use client';
+
+import Image from 'next/image';
+import { useState } from 'react';
+
+import {
+  avatarImage,
+  card,
+  cardContent,
+  cardFooter,
+  fallbackImage,
+  professorContact,
+  professorEmail,
+  professorName,
+  professorType,
+} from '../styles/professorCard.css';
+
+import type { Professor } from '../mocks/professor';
+
+function ProfessorCard({ professor }: { professor: Professor }) {
+  const [imgError, setImgError] = useState(false);
+
+  return (
+    <div className={card}>
+      {!imgError ? (
+        <img
```

Comment:
```suggestion
        <Image
```

Next.js Image 태그를 사용해주세요

Reply:
수정 완료했습니다!